e2e/app/eoa/testdata/threshold_reference.json

```diff
@@ -2,8 +2,8 @@
  "mainnet": {
   "ETH": {
    "cold": {
-    "min": "2.0480",
-    "target": "8.4800"
+    "min": "3.0480",
```

Comment:
How can these changes be explained? These tests have not much documents.

Reply:
see `dynamicThreshold`